### PR TITLE
39: Updated isLinear javadocs.

### PR DESCRIPTION
**Pull request by thodorisbais**
I'm not sure whether the javadoc provided fits our needs, but given that #39 is still important for 2.0, I believe we can always easily adjust the javadocs once we have an open PR :-) .

That said, I'm looking forward to your feedback.

---
This change is [Reviewable](https://reviewable.io/reviews/unitsofmeasurement/unit-api/181)

### CODE REVIEW

**Comment by keilw** [comment]
@thodorisbais Thanks, it looks OK but maybe @desruisseaux or @andi-huber could you have a second look at it if you get the chance?

**Comment by thodorisbais** [comment]
Maybe we merged it a bit early, but I'm also curious on Andy's and Martin's feedback.

**Comment by andi-huber** [comment]
To be more specific and relevant, how about this ...

```java

  /**
   * Indicates whether this converter represents a one-dimensional linear transformation.
   * <p>
   * Given such a 'linear' converter 'A', let 'u' and 'v' be arbitrary numbers and 
   * 'r' be a constant number, then the following must hold by definition: 
   *
   * <ul>
   * <li>{@code A(u + v) == A(u) + A(v)}</li>
   * <li>{@code A(r * u) == r * A(u)}</li>
   * </ul>
   * </p>
   *
   * <p>
   * Given a second 'linear' converter 'B', commutativity can be derived from above definition:
   *
   * <ul>
   * <li>{@code (A o B) (u) == (B o A) (u)}</li>
   * </ul>
   * 
   * In other words, two 'linear' converters do have the property that A(B(u)) == B(A(u)), meaning 
   * for A and B the order of composition does not matter.  
   * 
   * </p>
   *
   * @return {@code true} if this converter represents a one-dimensional linear transformation; 
   * {@code false} otherwise.
   */
boolean isLinear();
```

**Comment by keilw** [comment]
Thanks @andi-huber could you propose another PR with the suggested clarifications?
